hedera-mirror-grpc/src/main/java/com/hedera/mirror/grpc/repository/TopicMessageRepositoryCustom.java

```diff
@@ -25,4 +25,7 @@ public interface TopicMessageRepositoryCustom {
 
     @Transactional(readOnly = true)
     Stream<TopicMessage> findByFilter(TopicMessageFilter filter);
+
+    @Transactional(readOnly = true)
+    Stream<TopicMessage> findLatest(long consensusTimestamp, int limit);
```

Comment:
Think a list would be less overhead than stream since we're not paging through the results.
```suggestion
    List<TopicMessage> findLatest(long consensusTimestamp, int limit);
```